modules/mobile/spec/models/appointment_spec.rb

```diff
@@ -3,17 +3,27 @@
 require 'rails_helper'
 
 RSpec.describe Mobile::V0::Appointment, type: :model do
-  describe '.toggle_non_prod_id!' do
-    it 'toggles mocked ids to real ones' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('983')).to eq('442')
+  describe '.convert_non_prod_id!' do
+    it 'converts mocked ids to real ones' do
+      expect(Mobile::V0::Appointment.convert_non_prod_id!('983')).to eq('442')
     end
 
-    it 'toggles real ids back to mocked ones' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('442')).to eq('983')
+    it 'does not change id when already valid' do
+      expect(Mobile::V0::Appointment.convert_non_prod_id!('442')).to eq('442')
     end
 
     it 'keeps secondary identifiers' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('983GC')).to eq('442GC')
+      expect(Mobile::V0::Appointment.convert_non_prod_id!('983GC')).to eq('442GC')
+    end
+
+    context 'when not in production' do
+      before { Settings.hostname = 'api.va.gov' }
```

Comment:
I would consider doing this similarly to how we change the path for VCRs by first caching the value, then changing it back to the cached value. 

```
before do
  @original_hostname = Settings.hostname
  Settings.hostname = 'api.va.gov'
end

after { Settings.hostname = @original_hostname }
```

Just to avoid hard-coding a value that comes from a settings file. This is a bit of an academic point, though, because it's unlikely we'll ever change the specs hostname value and I'm pretty sure it's hard-coded in specs in a number of places. So feel free to ignore this comment.

Reply:
good point.